### PR TITLE
2809-2795 Add missing homologies and paralogs

### PR DESCRIPTION
**Pull request by ehsanb29**

### CODE REVIEW

**Review by mbdebian — CHANGES_REQUESTED** [review]
Good progress @ehsanb29 !
Please, have a look at the requested changes and re-submit the PR review request when done.

File: project/Dependencies.scala
[the remark this answers is not in the document]
Comment (ehsanb29) [reply project/Dependencies.scala:1]: Oh, I forgot to ignore it.

**Review by ehsanb29 — COMMENTED** [review]
All the changes is done i the next commit.

**Review by mbdebian — APPROVED** [review]
Great work!!! 🥳

File: configuration/2023/23_04_platform.conf
Comment (mbdebian) [inline configuration/2023/23_04_platform.conf:1]: Let's take this file out

File: configuration/2023/23_04_platform_workflow.conf
Comment (mbdebian) [inline configuration/2023/23_04_platform_workflow.conf:1]: Let's take this file out

**Review by mbdebian — APPROVED** [review]
🥳 Great work! 🥳

**Pull request merged**